doc/source/user_guide/timeseries.rst

```diff
@@ -2220,6 +2220,26 @@ you can use the ``tz_convert`` method.
 
    rng_pytz.tz_convert('US/Eastern')
 
+.. note::
+
+    .. versionadded:: 1.1.0
+
+    For ambiguous times, pandas supports explicitly specifying the keyword-only fold argument.
+    Due to daylight saving time, one wall clock time can occur twice when shifting
+    from summer to winter time; fold describes whether the datetime-like corresponds
+    to the first (0) or the second time (1) the wall clock hits the ambiguous time.
+    Fold is supported only for constructing from naive ``datetime.datetime`` or :class:`Timestamp`
+    or for constructing from components (see below) with ``dateutil`` timezones as ``pytz``
```

Comment:
if you can add a refernce to datetutil / pytz would be helpful here

Reply:
Done. I wasn't able to find a lot about `fold` treatment in the `dateutil` docs themselves, so I pointed to the functions that deal with `fold` and ambiguous times in the reference instead. `pytz` doesn't have anything about fold in the docs (obviously), but there is plenty about dealing with ambiguous times in the docs, so pointed there.